Tests/Unit/MSFT_WebApplicationHandler.tests.ps1

```diff
@@ -369,6 +369,18 @@ try
                 }
             }
         }
+
+        Describe 'MSFT_WebApplicationHandler/Export-TargetResource' {
+
+            Context 'Export Configuration' {
+
+                Mock Get-WebConfigurationProperty -MockWith {$mockCompliantHandler}
+
+                It 'Should Export all instances' {
+                    Export-TargetResource
+                }
```

Comment:
Thanks for adding the test, we can improve on those a bit.  Every Pester test should have some type of assertion after a command or variable to assert the expected result.  For a function that returns an output, the best way to test it is to invoke it and then compare the output to ensure it returns as expected.  

For this test, based on the mock Get-WebConfigurationProperty return `$mockCompliantHandler`, what would we expect the output string for Export-TargetResource to look like?  We can store that string in the test as `$webApplicationHandlerConfiguration` and assert the Export-TargetResource returns it.

For example:
```powershell
Mock -CommandName New-Guid -MockWith { [GUID]::Empty }

$webApplicationHandlerConfiguration = @"
WebApplicationHandler  00000000-0000-0000-0000-000000000000
{
    Path = 'MACHINE/WEBROOT/APPHOST'
    Name = 'ATest-WebHandler'
    PhysicalHandlerPath = '*'
    Verb = '*'
    Modules = 'IsapiModule'
    RequireAccess = 'None'
    ScriptProcessor = "C:\Windows\Microsoft.NET\Framework64\v4.0.30319\aspnet_isapi.dll"
    ResourceType = 'Unspecified'
    AllowPathInfo = $false
    ResponseBufferLimit = 0
    Type = $null
    PreCondition = $null
    Location = 'Default Web Site/TestDir'
}
@"

It 'Should return expected configuration' {
    Export-TargetResource | Should -Be $webApplicationHandlerConfiguration
}
```

Let's do that for all the Export-TargetResource tests.  At least one, but we could probably come up with multiple scenarios for some of these.  Like for xWebsite, what if zero websites return or multiple.  Will the Export-TargetResource return as expected?  Let me know if you need any help with that.